src/modm/driver/rtc/mcp7941_impl.hpp

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2021, Odin Holmes
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#ifndef MODM_MCP7941_HPP
+#	error  "Don't include this file directly, use 'mcp7941.hpp' instead!"
+#endif
+
+// ----------------------------------------------------------------------------
+template < typename I2cMaster >
+modm::Mcp7941<I2cMaster>::Mcp7941(Data &data, uint8_t address)
+:	I2cDevice<I2cMaster, 2>(address), data(data)
```

Comment:
```suggestion
:	I2cDevice<I2cMaster, 5>(address), data(data)
```

Does this driver already work on actually hardware? If not, try to increase the resumable depth...

Reply:
it does communicate 'something somewhat plausible' on real hardware. I will check that its the 'right'  thing. thanks for the tip though